### PR TITLE
feat(window): add `setEffects` API

### PR DESCRIPTION
**Pull request by lucasfernog**
Ref https://github.com/tauri-apps/tauri/pull/6442

Needs alpha.10 :)

### CODE REVIEW

File: examples/api/src-tauri/src/lib.rs [hunk 1]
```diff
@@ -71,6 +71,11 @@ pub fn run() {
                     .decorations(false);
             }
 
+            #[cfg(target_os = "macos")]
```
Comment (amrbashir) [inline examples/api/src-tauri/src/lib.rs:74]: Why not windows as well?

File: plugins/window/guest-js/index.ts [hunk 2]
```diff
@@ -900,6 +900,30 @@ class WindowManager extends WebviewWindowHandle {
...
+      value: effects,
+    });
+  }
+
+  /**
+   * Clear any applied effects if possible.
+   *
+   * @since 2.0
+   */
+  async clearEffects(): Promise<void> {
```
Comment (amrbashir) [inline plugins/window/guest-js/index.ts:970]: Should we add an explicit clear effects method on tauri's side? Instead of relying on None?

**Review by amrbashir — APPROVED** [review]

File: examples/api/src-tauri/src/lib.rs (hunk 1 above)
Comment (lucasfernog) [reply examples/api/src-tauri/src/lib.rs:74]: Windows is already handled on the block above.

File: plugins/window/guest-js/index.ts (hunk 2 above)
Comment (lucasfernog) [reply plugins/window/guest-js/index.ts:970]: I think setting None is quite idiomatic for Rust isn't it? For JavaScript it looks weird..

**Pull request merged**